app-libs/stf/src/trusted_call.rs

```diff
@@ -1232,8 +1232,19 @@ fn may_execute(tcs: &TrustedCallSigned) -> bool {
 				_ => false,
 			}
 		}
-	} else {
-		warn!("failed to apply call filter because mirrored shard config not available");
+	}
+	if MinimalChainSpec::is_known_production_chain(
+		shielding_target_genesis_hash().unwrap_or_default(),
+	) {
+		if matches!(
+			tcs.call,
+			TrustedCall::waste_time(..)
+				| TrustedCall::note_bloat(..)
+				| TrustedCall::spam_extrinsics(..)
```

Comment:
Good call!

Reply:
yeah. they are permissioned, but just protected with a private key. better to filter